VMBackup/main/Utils/WAAgentUtil.py

```diff
@@ -53,17 +53,27 @@ def searchWAAgentOld():
 pathUsed = 1 
 try:
     agentPath = searchWAAgent()
-    if(agentPath):
-        waagent = imp.load_source('waagent', agentPath)
+    if agentPath is None:
+       pathUsed = 0
+       # Search for the old agent path if the new one is not found
+       agentPath = searchWAAgentOld()
+    if agentPath:
```

Comment:
is this a null/empty check?

Reply:
yes, if it is not None. Similar to the one which were using previously